src/containers/Home/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Toaster } from 'react-hot-toast';
+import ProductList from './components/ProductList';
+import ProductForm from './components/ProductForm';
+
+const Home = () => {
+  return (
+    <div>
```

Comment:
```suggestion
    <>
```
Quoting what @hassantanveer97 told me

> "We can use React fragments where we need a single container for our components to reduce nesting in our components tree."